static/js/util/profile_edit.js

```diff
@@ -299,9 +317,25 @@ export function boundGeosuggest(
     R.unary(hasAllAddressProps),
   ]);
 
+
+  //Retrieve an profile's property value given a property path.
+  const getPath = (addressKey) => (R.path(addressKey, profile));
+
+  // Get all the values of a location (city, state, country) as an array.
+  // Convert state and country elements to ISO3166 codes
+  const getLocationElements = () => {
+    let elements = R.map(getPath, R.values(addressMapping));
```

Comment:
we could replace `getPath` here with this:

```js
R.path(R.__, profile)
```

which is just a little more compact (`R.__` is like an argument placeholder, so that function already has `profile` bound as its second argument and is waiting for its first)